montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -34,6 +38,23 @@ def load(self, solver, **kwargs):
 
         na, nbl, ntime, nchan = solver.na, solver.nbl, solver.ntime, solver.nchan
 
+	uvw_order = kwargs.get('uvw_order', ORDER_OTHER)
+        if uvw_order not in VALID_ORDERINGS:
+                raise ValueError('Invalid UVW ordering %s', uvw_order)
+
+        # Define transpose axes to convert file uvw order 
```

Comment:
@marziarivi Please convert all tabs in the file to 4 space characters. The indents are out in some places.